### PR TITLE
feat: add dragnode info into tree#allowdrop

### PR DESCRIPTION
**Pull request by moonrailgun**
Pass `dragNode` info when check `allowDrop` which can help us to decide which case should be allow trigger drop event

### CODE REVIEW

File: src/util.tsx [hunk 1]
```diff
@@ -141,13 +141,15 @@ export function calcDropPosition(
     }
   }
 
+  const abstractDragDataNode = keyEntities[dragNode.props.eventKey].node;
```
Comment (07akioni) [inline src/util.tsx:144]: `dargNode` is passed from arguments, why get it from the map?

Comment (moonrailgun) [reply src/util.tsx:144]: The argument one is `NodeInstance`. and i think the useful one to user is `DataNode`

Comment (07akioni) [reply src/util.tsx:144]: can't we get datanode from nodeinstance?

Comment (07akioni) [reply src/util.tsx:144]: Yes, I think get it from arg but not map is clearer.

File: README.md [hunk 2]
```diff
@@ -138,4 +138,3 @@ rc-tree is released under the MIT license.
 - [jqTree](https://mbraak.github.io/jqTree/)
 - [jquery.treeselect](https://travistidwell.com/jquery.treeselect.js/)
 - [Angular Multi Select Tree](https://a5hik.github.io/angular-multi-select-tree/)
```
Comment (07akioni) [inline README.md:140]:
```suggestion
- [Angular Multi Select Tree](https://a5hik.github.io/angular-multi-select-tree/)

```

**Review by 07akioni — APPROVED** [review]

**Comment by moonrailgun** [comment]
Its looks like some ci workflow problem pending it, could you push it?
@07akioni @afc163

**Comment by 07akioni** [comment]
@afc163 done

File: README.md (hunk 2 above)
Comment (07akioni) [reply README.md:140]: @moonrailgun

**Pull request merged**